src/Microsoft.Health.Fhir.R4.Web/Microsoft.Health.Fhir.R4.Web.csproj

```diff
@@ -5,6 +5,16 @@
     <RootNamespace>Microsoft.Health.Fhir.Web</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Here and in the other .csproj files, the definitions.zip has a line for Remove= and another line for Include=.  That seems odd, perhaps we don't need the Remove=?

Reply:
That was added automatically when setting the definition.zip as "content" type in the file properties.

I will double check if everything works fine without it.